components/component.go

```diff
@@ -39,6 +39,10 @@ type Component struct {
 	DevFlags *DevFlags `json:"devFlags,omitempty"`
 }
 
+func (c *Component) Init(_ context.Context, _ cluster.Platform) error {
+	return nil
+}
```

Comment:
why need this? you foresee some component might not have implementation?

Reply:
Frankly put just in case. Can remove.

Reply:
thought was left from the first commit, when only one component implemented with Init(..) at that time.

Reply:
That is also true, I implemented it one by one and squashed later, but did not pay attention.

Reply:
lets keep it for now, i dont have strong opinion to remove it